core/templates/pages/library-page/search-bar/search-bar.directive.html

```diff
@@ -226,6 +226,13 @@
       display: none;
     }
   }
+
+  @media (max-width: 766px) {
+    .oppia-search-bar-category-selector {
```

Comment:
Wait, is this change part of this PR? Isn't this a separate PR? Can you remove these changes from here?

Reply:
sorry for my inconvenience, I did this mistake in haste.